app/views/saved_declarations.scala.html

```diff
@@ -39,20 +38,27 @@
 
     <h1>@messages("saved.declarations.title")</h1>
 
+    @helper.CSRF.formField
```

Comment:
Whys this here? the page doesn't have a form

Reply:
I have not mastered the dark art of `copy and paste`

Reply:
This has been addressed.